### PR TITLE
First commit to SecondBranch

### PR DESCRIPTION
**Pull request by Bronzza**

### CODE REVIEW

File: src/main/java/applicationPackage/Application.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class Application {
+    private final static String DECODED_FILE_NAME = "decod.txt";
+
+    public static void main(String[] args) {
+        List<Map<List<Set<Integer>>, String>> list = createMultiLayoutList();
+        XMLEncoder xmlEncoder = null;
```
Comment (Taniann) [inline src/main/java/applicationPackage/Application.java]: better move the logic of xml encoding to the separate util method

File: src/main/java/applicationPackage/Application.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class Application {
+    private final static String DECODED_FILE_NAME = "decod.txt";
+
+    public static void main(String[] args) {
```
Comment (Taniann) [inline src/main/java/applicationPackage/Application.java:17]: better move main method to the separate app class

**Review by Taniann — CHANGES_REQUESTED** [review]
resolve comments + add logger

File: src/main/java/applicationPackage/Application.java (hunk 2 above)
Comment (Bronzza) [reply src/main/java/applicationPackage/Application.java:17]: What should I do with the main method class. Should I remove all the methods from this class beside mainMethod? Even a technical method to create a list to decode?

File: src/main/java/xmlUtils/XmlUtils.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.Setter;
+import org.apache.log4j.Logger;
+
+import java.beans.XMLEncoder;
+import java.io.BufferedOutputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+
+public class XmlUtils {
+    @Getter
```
Comment (Taniann) [inline src/main/java/xmlUtils/XmlUtils.java]: why do you need setter and getter for logger?

**Review by Taniann — APPROVED** [review]

**Pull request merged**